test/src/CMakeLists.txt

```diff
@@ -19,18 +19,27 @@ SET_TARGET_PROPERTIES(1-sec-prog
 ADD_EXECUTABLE(leak-tiny_32 leak-tiny.c)
 ADD_EXECUTABLE(leak-huge_32 leak-huge.c)
 ADD_EXECUTABLE(leak-dive_32 leak-dive.c)
+ADD_EXECUTABLE(leak-data_32 leak-data.c)
+ADD_EXECUTABLE(leak-bss_32  leak-bss.c)
 SET_TARGET_PROPERTIES(leak-tiny_32
                       leak-huge_32
                       leak-dive_32
+                      leak-data_32
+                      leak-bss_32
                       PROPERTIES COMPILE_FLAGS "-m32"
                                  LINK_FLAGS "-m32")
 ADD_EXECUTABLE(leak-tiny_64 leak-tiny.c)
 ADD_EXECUTABLE(leak-huge_64 leak-huge.c)
 ADD_EXECUTABLE(leak-dive_64 leak-dive.c)
+ADD_EXECUTABLE(leak-data_64 leak-data.c)
+ADD_EXECUTABLE(leak-bss_64  leak-bss.c)
 SET_TARGET_PROPERTIES(leak-tiny_64
                       leak-huge_64
-                      leak-dive_64 PROPERTIES COMPILE_FLAGS "-m32"
-                                              LINK_FLAGS "-m32")
+                      leak-dive_64
+                      leak-data_64
+                      leak-bss_64
+                      PROPERTIES COMPILE_FLAGS "-m32"
+                                 LINK_FLAGS "-m32")
```

Comment:
Skoro tu jest `-m32`, to czym się różni _32 od _64 ?

Reply:
Hm, znalazłeś buga w testach :)